lib/debt_to_income_ratio_latest/main.py

```diff
@@ -96,25 +103,25 @@ async def get_debt_to_income_ratio_latest(
     if len(depository_credit_records_list) == 0:
         return float("inf")
 
-    # Here we sort by timestamp so latest balances are on top
+    # STEP 6: Here we sort by timestamp so latest balances are on top
     loan_records_list = sorted(loan_records_list, key=lambda x: x["ts"], reverse=True)
```

Comment:
I was trying to think if there was a different way to write this that might be "easier" to reproduce in other languages but I can't think of one. Any ideas or think it translates well?

Reply:
Me too. Making it readable would imply doing a custom sorting instead of using `sorted` I guess. All modern languages have generic sorting features with something similar to lambdas for custom criteria, so I guess that it is ok as we have really just a couple of these structures around.